components/CollectiveNavbar.js

```diff
@@ -304,6 +305,15 @@ export const getFilteredSectionsForCollective = (collective, isAdmin, isHostAdmi
   if (!collective.settings?.recurringContributions) {
     toRemove.add(Sections.RECURRING_CONTRIBUTIONS);
   }
+  // don't display for TYPE=COLLECTIVE if no active contributions
+  if (collective.type === CollectiveType.COLLECTIVE) {
```

Comment:
Let's mention that in demos, maybe we also want it for Organizations and Individuals.